WhaleSpotting/react-app/src/styles/Constants.scss

```diff
@@ -38,7 +38,7 @@ $guttersSide: 20px;
 $minMobileScreenSize: 320px;
 $mobileScreenSize: 445px;   
 
-$headerHeight: 6vh;
+$headerHeight: 40px;
```

Comment:
So if we are changing this to px, we should also probably change the footer and mobile styling. I am not sure this needs to be px, though, does this make the styling nicer?

Reply:
I personally believe so, it makes everything more precise.